src/android/Capture.java

```diff
@@ -369,10 +406,8 @@ else if (resultCode == Activity.RESULT_CANCELED) {
 
 
     public void onAudioActivityResult(Request req, Intent intent) {
```

Comment:
can remove the `Intent intent` for the 3 `onAudio/Image/VideoActivityResult` not used anymore